### DISCUSSION TITLE
How to load file from known uuid/uuids

### DISCUSSION THREAD

**Discussion by razor2599** [uploadcare/file-uploader#429] [blocks] [answered]
Hi!
Instead of uploading files, I want to load files with known uuids. Is it possible?

<lr-file-uploader-minimal uuid="2efb27fd-f631-431c-988b-7f21eb57dc49"
    css-src="https://unpkg.com/@uploadcare/blocks@0.15.1/web/file-uploader-minimal.min.css">
</lr-file-uploader-minimal>

**Comment by loookashow** [comment]
Hello, @sinuho. Could you please provide a detailed description of your task? This would enable us to identify a potential solution.

**Comment by nd0ut** [comment, marked as answer]
Hi there, 

Unfortunately, we don't have the exact feature you're looking for. However, we do have a workaround that could help you. Check out this link: https://codesandbox.io/s/gallant-nightingale-h03q8h?file=/index.html 

And as @loookashow said - it would be awesome if you could tell us more about your specific case. This way, we can create an even better API that covers your needs.

**Comment by ha-sante** [comment]
@nd0ut @loookashow 


This is what I see when I upload an image.
![image](https://github.com/uploadcare/blocks/assets/90225652/59518551-b918-48cd-9d29-1d6cc39aa057)

When I load/move to a different part of the page, this is cleared.
![image](https://github.com/uploadcare/blocks/assets/90225652/388e5b9e-b21a-4c21-b7fd-1d34604f579c)

- I would like the ability to set the current image uploader to a URL I provide, without it triggering a new upload event, but showing this uploaded already effect.

**Comment by nd0ut** [comment]
Hey @ha-sante, got it. We've replaced the `addFiles` method with separate methods: `addFileFromUrl`, `addFileFromUuid`, and `addFileFromObject`. These new methods provide the option to make the process silent, without triggering the upload event. The updated version will be released in a few days.

**Reply by ha-sante** [reply]
Okay, that's good news to hear. I will be looking forward to it so I can change to that. This is highly appreciated!

Thank you guys for all the work you do.  🙏

**Reply by nd0ut** [reply]
Here it is: https://github.com/uploadcare/blocks/releases/tag/v0.23.0
